source/extensions/filters/http/oauth2/filter.h

```diff
@@ -85,9 +85,10 @@ class SDSSecretReader : public SecretReader {
  * All stats for the OAuth filter. @see stats_macros.h
  */
 #define ALL_OAUTH_FILTER_STATS(COUNTER)                                                            \
-  COUNTER(oauth_unauthorized_rq)                                                                   \
+  COUNTER(oauth_new)                                                                               \
   COUNTER(oauth_failure)                                                                           \
-  COUNTER(oauth_success)
+  COUNTER(oauth_success)                                                                           \
+  COUNTER(oauth_unauthorized)
```

Comment:
given the filter's status is alpha, I presume it's fine to break stats

cc: @snowp